pandas/core/reshape/merge.py

```diff
@@ -1083,7 +1067,7 @@ def _get_join_indexers(left_keys, right_keys, sort=False, how='inner',
 
 
 class _OrderedMerge(_MergeOperation):
-    _merge_type = 'ordered_merge'
+    _merge_type = 'merge_ordered'
```

Comment:
This one should not be changed I think. `_merge_type` is passed to `__finalize__`, so is mainly meant for subclasses being able to do something special. So we can leave that intact (only very advanced use case anyhow, it is not visible to normal user)

Reply:
I don't understand this either: ``ordered_merge`` is removed, so calling it in that ``finanlize`` would cause an AttributeError. What am I missing here?

Reply:
`ordered_merge` and `merge_ordered` where just aliases for the same underlying code, so both have been using `_merge_type = ordered_merge`. Changing this could break subclasses